server/assets-v1/templates/src/pages/client_portal/register.html

```diff
@@ -187,18 +185,48 @@ <h1 class="font-bold text-3xl md:text-4xl text-[#3751FE] mb-11 animate-slideFrom
               password: password.value,
             }),
           });
-          alert("Registration successful!");
+          showToastMessage("Registration successful!", "error")
           window.location.href = "[[ .ProxyURL ]]/client-login-page";
         } catch (error) {
           console.error(error);
-          alert("Registration failed!");
+          showToastMessage("Registration failed!", "error")
+        }
+      };
+
+      const showToastMessage = (message, type) => {
+        toastMessage.value = message;
+        showToast.value = true;
+        setTimeout(() => {
+          showToast.value = false;
+        }, 3000);
+      };
+
+      const checkBackendUri = async () => {
```

Comment:
Is it only showing toast, is users still able to submit after entering duplicated backend url?

Reply:
Backend URL is required. If user enter dublicated backend url, backend url input will be empty. Then, system ask backend url.